primitives/consensus/beefy/src/lib.rs

```diff
@@ -303,15 +322,14 @@ impl OpaqueKeyOwnershipProof {
 }
 
 sp_api::decl_runtime_apis! {
-	/// API necessary for BEEFY voters.
-	#[api_version(2)]
-	pub trait BeefyApi
+	/// API necessary for BEEFY voters with only ECDSA key.
+	pub trait BeefyApi<AuthorityId> where AuthorityId : Encode + Decode
```

Comment:
silly nit (feel free to ignore)

```suggestion
	pub trait BeefyApi<AuthorityId> where AuthorityId : Codec
```